src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * 40;
+  const longTerm = 7;
```

Comment:
```suggestion
  const LONG_TERM = 7;
```
<img width="703" alt="Screenshot 2023-10-30 at 14 13 28" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/135232001/562ac8b4-0504-4884-9d43-bce5600a5e42">